docs/core/responses.rst

```diff
@@ -38,6 +39,32 @@ you need to retrain the assistant before these changes will be picked up.
 More details about the format of these responses can be found in the
 documentation about the domain file format: :ref:`utter_templates`.
 
+.. _open-domain-utterances:
+
+Managing responses to open domain intents
+-------------------------------------------
+
+All assistant utterances which should be used as a reply to open domain intents can be included as part of ``common examples`` in
+the NLU training data file.
+
+For example -
+
+.. code-block:: md
+
+    ## intent: faq, response: The supported Python versions are: 2.7,3.5,3.6. The recommended version is 3.6.
```

Comment:
This would be considered as a usual training example for intent classification and not considered as an open domain intent.
Although, I see your point that we should add a validation while loading training data that all training examples for open domain intents should have a `response` block necessarily.